TripPlaner/src/main/java/com/singapore/TripPlaner/Model/Places.java

```diff
@@ -8,16 +8,16 @@ public class Places /*extends PersistentAbstract*/ {
     private double rate;
     private String opinion;
     private City city;
-
-    private long fromCity;
-
-    public long getFromCity() {
-        return fromCity;
-    }
-
-    public void setFromCity(long fromCity) {
-        this.fromCity = fromCity;
-    }
+//
+//    private long fromCity;
+//
+//    public long getFromCity() {
+//        return fromCity;
+//    }
+//
+//    public void setFromCity(long fromCity) {
+//        this.fromCity = fromCity;
+//    }
```

Comment:
czemu zakomentowane ? mozna od razu usunąć, jak będzie potrzeba wrócić do tego to jest w historii gita.